package.json

```diff
@@ -20,7 +20,7 @@
     "updated": "lerna updated --only-explicit-updates",
     "prettier": "node prettier.js",
     "clean": "lerna clean --yes && rimraf .jest",
-    "publish": "git pull --rebase && check-installed-dependencies && npm run build && lerna publish --only-explicit-updates",
+    "publish": "git pull --rebase && check-installed-dependencies && npm run build && lerna publish",
```

Comment:
Revert this please. We don't want half a dozen packages with no code changes getting a version bump just because some util they all depend on get one.